libminifi/include/utils/StringUtils.h

```diff
@@ -36,7 +36,11 @@ enum TimeUnit {
   MILLISECOND,
   NANOSECOND
 };
-
+#if (__cplusplus >= 201103L && (!defined(__GLIBCXX__) || (__cplusplus >= 201402L) ||  (defined(_GLIBCXX_RELEASE) && _GLIBCXX_RELEASE > 4)))
```

Comment:
If I remember well that's gonna fail for Win given the way MS handled __cplusplus before:
https://devblogs.microsoft.com/cppblog/msvc-now-correctly-reports-__cplusplus/